File: CodeFreak1/ClientApp/src/app/Security/Dtos/permissions-view-model.ts
import { RequestStatus } from "./request-status";

export class PermissionsViewModel extends RequestStatus {
  public PermissionId: number;
  public Name: string;
  public Description: string;
  public CreatedBy: number;
  public CreatedOn: Date;
  public Modifiedby: string;
  public ModifiedOn: Date;
  public IsActive: boolean;
}

File: CodeFreak1/ViewModel/PermissionsViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CodeFreak1.ViewModel
{
    public class PermissionsViewModel:RequestStatus
    {

        public int PermissionId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public Guid? Modifiedby { get; set; }
        public DateTime? ModifiedOn { get; set; }
        public bool? IsActive { get; set; }

    }
}
